README.md

```diff
@@ -45,26 +45,27 @@ import { Route, Switch } from 'react-router';
 ```
 *V4 Notes:*
 
-In React Router v4 a `<Route>` can no longer have children *and* a component so replacing them above with `{this.props.children}` will not work. If you are unaware of what `Route.render` is it is advised you [read about it here](https://reacttraining.com/react-router/web/api/Route/render-func).
+In React Router v4 a `<Route>` can no longer have children *and* a component so replacing them above with `{this.props.children}` will not work. If you are unaware of what `Route.render` does, [read about it here](https://reacttraining.com/react-router/web/api/Route/render-func).
 
 ### Installation
 
 `npm install --save react-router-transition`
 
 ### Usage
 
-`RouteTransition` requires a few props:
-- `pathname`: the key signifying the transitionable route, most typically the pathname (required)
+#### `<RouteTransition />`
```

Comment:
i actually want to tuck away `RouteTransition`, and only expose it as a low-level wrapper for `TransitionMotion`, should `AnimatedSwitch` or `AnimatedRoute` not be enough.